pom.xml

```diff
@@ -11,74 +11,181 @@
         <graalvm.version>1.0.0-rc8</graalvm.version>
         <compiler.dir>${project.build.directory}/compiler</compiler.dir>
     </properties>
+    <profiles>
+        <profile>
+            <id>jdk11</id>
+            <activation>
+                <jdk>11</jdk>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.graalvm.sdk</groupId>
+                    <artifactId>graal-sdk</artifactId>
+                    <version>${graalvm.version}</version>
```

Comment:
GraalVM has `graal-sdk` on bootclasspath. But still, shouldn't this be added into the common dependencies section? If somebody opens this project on regular JDK8, then it will be full of errors. If you move `graal-sdk` into regular dependency outside of `profiles`, the project will build on any JDK8.

Reply:
We don't want to support JDK8 as it will not support JS compilation.

Reply:
How do you prevent users to open the project in an IDE that is using JDK8 as the Java platform? Once they do it, they see tons of compilation errors. E.g. it is not about running the sample, but about compiling it. 

It would, in my opinion, be better if the example could be compiled on any JDK8.

Of course, the execution can fail saying that "GraalVM CE" should be downloaded. Btw. in other projects I am using a plugin to open a browser when not running on GraalVM CE - like [here](https://github.com/JaroslavTulach/matrixultimate/blob/master/pom.xml#L161).

Reply:
I'd say the the compilation errors are a feature not a bug :) i.e. they are an indication that their IDE is not properly set up.